lib/ir_pagerduty.rb

```diff
@@ -0,0 +1,19 @@
+require 'pagerduty/full'
+
+class IRPagerduty < PagerDuty::Full
+  def initialize
+    @apikey = SupportApp.pager_duty_token
+    @subdomain = SupportApp.pager_duty_subdomain
```

Comment:
Rather than setting instance variables we should call the superclass constructor:

```
  def initialize
    super(SupportApp.pager_duty_token, SupportApp.pager_duty_subdomain)
  end
```
```